gradle/spotless.gradle

```diff
@@ -31,3 +31,9 @@ spotless {
 		greclipse().configFile rootProject.files('gradle/spotless.eclipseformat.xml', 'gradle/spotless.groovyformat.prefs')
 	}
 }
+tasks.named('build') {
+	dependsOn 'spotlessApply'
```

Comment:
how to disable this on cloud?

it should not run there, as waste of time thus potentially cause issues.